python/ql/src/experimental/Security/CWE-287-ConstantSecretKey/examples/config3.py

```diff
@@ -0,0 +1,7 @@
+"""Flask App configuration."""
+
+# General Config
+FLASK_DEBUG = True
+SECRET_KEY = "CHANGEME5"
+if SECRET_KEY == "CHANGEaME5":
+    raise "not possible"
```

Comment:
Is this really safe? Does it not just give you another constant to keep up-to-date?

Reply:
Ah, so I guess line 5 is not in production code and line 6 compares against a known default constant?
They still have to keep line 6 up-to-date, though, if defaults change, right?